src/shared-components/Form/Control/Slider.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box } from '@mui/material';
+import Slider from '@mui/material/Slider';
+import { theme } from 'src/styles/theme';
+
+interface CustomSliderProps {
+  defaultValue: number;
+  color: string;
+  thumbbgcolor: string;
+  trackcolor: string;
+  railcolor: string;
+}
+
+export const CustomSlider = ({
+  defaultValue,
+  color,
+  thumbbgcolor,
+  trackcolor,
+  railcolor,
+}: CustomSliderProps) => {
+  return (
+    <Box sx={{ width: 343 }}>
```

Comment:
Don't add this wrapper around the slider, let it be as wide as the parent element allow.

Reply:
Done.